src/content/changelog/4-11-1-2019-12-23.md

```diff
@@ -0,0 +1,11 @@
+---
+title: '4.11.1: 2020-01-13'
+date: 2020-01-13T05:00:00.000Z
+version: 4.11.1
+changelog:
+  - API
+---
+_Fixes_
```

Comment:
This is italicized in the output, which doesn't match the style of the earlier entries.

Reply:
Fixed in latest commit